hwc3-service/default/Android.bp

```diff
@@ -0,0 +1,86 @@
+//  Copyright (C) 2022 The Android Open Source Project
+// 
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+// 
+//       http://www.apache.org/licenses/LICENSE-2.0
+// 
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+// package {
+//     // See: http://go/android-license-faq
+//     // A large-scale-change added 'default_applicable_licenses' to import
+//     // all of the 'license_kinds' from "hardware_interfaces_license"
+//     // to get the below license kinds:
+//     //   SPDX-license-identifier-Apache-2.0
+//     default_applicable_licenses: ["hardware_interfaces_license"],    
+// }
+
+cc_binary {
+    name: "android.hardware.graphics.composer3-service.intel",
+	relative_install_path: "hw",
+    init_rc: ["android.hardware.graphics.composer3-service.intel.rc"],
+    vintf_fragments: ["android.hardware.graphics.composer3-service.intel.xml"],
+    vendor: true,
+    cflags:[
+        "-Wall",
+        "-Werror",
+        "-DLOG_TAG=\"hwc3\"",
+	],
+    cppflags: [
+        "-Wextra",
+        "-Wunused",
+        "-Wunreachable-code",
+        "-Wno-sign-compare",
+    ],
+	local_include_dirs:[
+		"include",
+        "impl",
+	],
+    include_dirs: [ 
+    ],
+
+    header_libs: [
+        "android.hardware.graphics.composer@2.1-passthrough",
```

Comment:
double check it

Reply:
don't need android.hardware.graphics.composer@2.1-passthrough